packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -724,23 +752,19 @@ class _StretchingOverscrollIndicatorState extends State<StretchingOverscrollIndi
           final double stretch = _stretchController.value;
           double x = 1.0;
           double y = 1.0;
-          final AlignmentDirectional alignment;
+          AlignmentDirectional? alignment;
```

Comment:
It probably doesn't matter in this case as Transform can take a nullable alignment, but this might be better expressed as:

```dart
  late final AlignmentDirectional alignment;
```

As we should always have a valid alignment here, no?